scripts/update_firefox_os_feeds.py

```diff
@@ -0,0 +1,36 @@
+from django.conf import settings
+
+from feedparser import parse
+
+from bedrock.firefox.models import FirefoxOSFeedLink
+
+
+def create_or_update_fxos_feed_link(locale, link, title):
+    try:
+        feed_link = FirefoxOSFeedLink.objects.get(
+            locale=locale, link=link)
+    except FirefoxOSFeedLink.DoesNotExist:
+        FirefoxOSFeedLink.objects.create(locale=locale, link=link,
+                                         title=title)
+    else:
+        if title != feed_link.title:
+            feed_link.title = title
+            feed_link.save(update_fields=['title'])
+
+
+def create_or_update_fxos_feed_links(locale, parsed_feed):
+    for entry in parsed_feed.get('entries', []):
+        link = entry.get('link')
+        title = entry.get('title')
+        if link and title:
+            create_or_update_fxos_feed_link(locale, link, title)
+
+
+def run(*args):
+    for locale, url in settings.FIREFOX_OS_FEEDS:
+        try:
+            parsed_feed = parse(url)
+        except:
+            print ('Unable to fetch and parse ' + url)
```

Comment:
I guess... but they look silly with the space and will be wrong pep8-wise when we do switch right? Be better to be explicit and just import `print_function` from `__future__` don't you think?